### PR TITLE
Calc errors

### PR DESCRIPTION
**Pull request by Bjvanminnen**
Adds a couple new error messages for Calc (and adds one of these to Eval as well).

These messages happen before animation, and don't end up with the svg in the error message.
image

### CODE REVIEW

File: apps/src/StudioApp.js [hunk 1]
```diff
@@ -1308,3 +1308,20 @@ StudioApp.prototype.updateHeadersAfterDropletToggle_ = function (usingBlocks) {
 StudioApp.prototype.hasExtraTopBlocks = function () {
   return this.feedback_.hasExtraTopBlocks();
 };
+
+/**
+ * @param {Blockly.Block} block Block to check
```
Comment (bcjordan) [inline apps/src/StudioApp.js]: block block block :)

File: apps/src/calc/calc.js [hunk 2]
```diff
@@ -655,7 +693,7 @@ function cloneNodeWithoutIds(elementId) {
  * App specific displayFeedback function that calls into
  * studioApp.displayFeedback when appropriate
  */
-function displayFeedback() {
+function displayFeedback(includeAppDiv) {
```
Comment (bcjordan) [inline apps/src/calc/calc.js]: Is this unused?

**Comment by bcjordan** [comment]
LGTM

File: apps/src/calc/calc.js (hunk 2 above)
Comment (Bjvanminnen) [inline apps/src/calc/calc.js]: Yep. Got replaced by isPreAnimationFailure

**Pull request merged**